ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/GIT_Source/tasks/main.yml

```diff
@@ -54,13 +54,23 @@
     # TODO: Package installs should not use latest
     - skip_ansible_lint
 
-- name: Compile and install git from source
+- name: Compile and install git from source on RHEL/CentOS6
+  shell: cd /tmp/git-2.15.0 && ./configure --with-curl=/usr/local/curl-7.61.1 --prefix=/usr/local --without-tcltk && make clean && make -j {{ ansible_processor_vcpus }} && sudo make install
```

Comment:
Should we be using sudo everywhere or nowhere?  I am fine with either I only ask because I have a systems both ways and I think without sudo I cannot run the playbook directly on the machine, but via AWX its fine.

Reply:
Should be without as it's connecting as root ... That was copied from the equivalent other lina already in the file - can I suggest creating a separate issue for this discussion and we'll apply the outcome globally across the playbooks instead of just addressing it here?